unit-test/background/pixel.es6.js

```diff
@@ -1,5 +1,5 @@
 
```

Comment:
hm, since we changed the original file being testsed here, I think we should also rename the test file

Reply:
Yeah I was in two minds about this (as it's only testing pixels and not the whole thing), I think I agree :)